.github/workflows/test.yml

```diff
@@ -98,6 +99,9 @@ jobs:
             test-suite: 'test/hotspot/jtreg/:tier1_serviceability'
             debug-suffix: -debug
 
+          - test-name: 'lib-test/tier1'
+            test-suite: 'test/lib-test/:tier1'
```

Comment:
I think this one should be executed with `debug` builds, like the Hotspot testing? Because this touches some internal diagnostic interfaces, and thus would benefit from checking the internal asserts hold. Should be enough to add `debug-suffix` parameter here.

Reply:
fixed, thanks